app/views/home/index.html.erb

```diff
@@ -1,28 +1,29 @@
 <header class="jumbotron text-center bg-gray-light mb-6 border-bottom px-3">
   <div class="container-lg position-relative">
     <h1 class="alt-h1">Open Source Friday</h1>
-    <p class="alt-lead text-gray mb-md-5 col-md-8 mx-auto">Take some time one day every week to invest in the open source software you already use and the community of people that make it great</p>
+    <p class="alt-lead text-gray mb-md-5 col-md-8 mx-auto">Once a week, every week. Take time to invest in the open source software you use and love.</p>
   </div>
 </header>
 
 <div class="container-md py-6 py-3 px-3">
   <h2 class="alt-h2 text-purple text-center">Your business depends on it</h2>
 
   <p class="alt-lead text-gray col-md-11 mx-auto text-center mb-6">
-    Encourage your employees and coworkers to invest in the software you use and the community that makes it great.
+    Companies need open source software to run effectively.
   </p>
 
   <div class="col-sm-6 col-md-12 float-left float-md-none pr-sm-4 pr-md-0 mb-md-4 clearfix">
     <div class="col-12 col-md-6 float-md-left mb-4 pr-md-6">
-      <h3 class="alt-h3 mb-1">Improve your technology</h3>
+      <h3 class="alt-h3 mb-1">Attract and retain talent</h3>
       <p class="alt-text-small text-gray mb-4">
-        Open source infrastructure is essential to running businesses effectively. Take control rather than relying entirely on others to maintain the software your business depends on.
+        Employees that work on open source boost their, and their employer's, reputation.
       </p>
 
-      <h3 class="alt-h3 mb-1">Reduce overhead</h3>
+      <h3 class="alt-h3 mb-1">Work better together</h3>
       <p class="alt-text-small text-gray mb-4">
-        Regular contributions to open source saves you money by reducing the custom software you need to maintain.
+        Open source development models are at the forefront of software collaboration.
```

Comment:
Added a bit more InnerSourcey detail. It's not perfect, but we can polish later.